metabase/metabase
# Convert Tree component to Typescript

### Changes
This PR converts `Tree` component that is used in permissions views to Typescript 

### How to verify
- Ensure tests are passing
- Check the tree component manually on [the permissions page](http://localhost:3000/admin/permissions/data/group)
